src/main/java/wolox/training/helpers/Json.java

```diff
@@ -0,0 +1,14 @@
+package wolox.training.helpers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class Json {
```

Comment:
Is there a specific purpose for this class? I would use `new ObjectMapper()).readTree(jsonString` directly in the class that is currently using Json :)